ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -125,6 +125,12 @@ const CoinButtons = ({
   const account = useSelector(getSelectedAccount);
   const { address: selectedAddress } = account;
   const history = useHistory();
+  ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
+  const currentActivityTabName = useSelector(
+    // @ts-expect-error TODO: fix state type
+    (state) => state.metamask.defaultHomeActiveTabName,
```

Comment:
Can't we just type it properly now? IDK if this would work:

```suggestion
    (state: { metamask: AppStateControllerState }) => state.metamask.defaultHomeActiveTabName,
```

Reply:
Unfortunately not, its a merged metamask state